test/http2_test/test_max_streams.py

```diff
@@ -0,0 +1,31 @@
+import logging
+import http2_base_server
+from hyperframe.frame import SettingsFrame
+
+class TestcaseSettingsMaxStreams(object):
+  """
+    This test sets MAX_CONCURRENT_STREAMS to 1 and asserts that at any point
+    only 1 stream is active.
```

Comment:
Mentioned that all the assertions happen at the twisted layer. Otherwise its confusing when people look through the file and don't see any assertions.